Makefile

```diff
@@ -68,9 +72,12 @@ build-cli-mac-intel: build-injector-registry-amd build-ui ## Build the Zarf CLI
 build-cli-mac-apple: build-injector-registry-arm build-ui ## Build the Zarf CLI for macOS on ARM
 	GOOS=darwin GOARCH=arm64 go build -ldflags="$(BUILD_ARGS)" -o build/zarf-mac-apple main.go
 
+build-cli-windows-amd: build-injector-registry-amd build-ui
```

Comment:
We should be supporting Window and Windows on ARM if that's possible.

Reply:
Agreed, I did add it already to goreleaser, should just need to update the makefile.

Reply:
want to do some more makefile cleanup so I'll open a new PR for those changes + the arm make target